pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -50,6 +55,9 @@ func TestSingleZone(t *testing.T) {
 		{
 			ObjectMeta: metav1.ObjectMeta{Name: "Vol_Stable_2", Labels: map[string]string{v1.LabelTopologyRegion: "us-west1", "uselessLabel": "none"}},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "Vol_With_Region", Labels: map[string]string{v1.LabelTopologyZone: "us-west1-a", v1.LabelTopologyRegion: "us-west1-a"}},
```

Comment:
Maybe this:
```suggestion
			ObjectMeta: metav1.ObjectMeta{Name: "Vol_Stable_Region_Zone", Labels: map[string]string{v1.LabelTopologyZone: "us-west1-a", v1.LabelTopologyRegion: "us-west1-a"}},
```

or simply `Vol_Stable_3` to follow the pattern above?